.github/workflows/fortran.yml

```diff
@@ -0,0 +1,55 @@
+name: Fortran
+
+on:
+  push:
+    branches: 
+    - main
+  pull_request:
+    branches: 
+    - main
+jobs:
+  build-and-test-fortran:
+    name: Fortran ${{matrix.build}} ${{matrix.os}}
+    runs-on: ${{matrix.os}}
+    strategy:
+      fail-fast:
+      matrix:
+        build: ["Release", "Debug"]
+        os: [ubuntu-20.04]
+        llvm: [15]
+        include:
+          - llvm: 15
+            ifx: 2023.0.0
+            mpi: 2021.7.1
+    timeout-minutes: 60 
+    steps:
+    - name: add llvm
+      run: |
+        wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
+        sudo apt-add-repository "deb http://apt.llvm.org/`lsb_release -c | cut -f2`/ llvm-toolchain-`lsb_release -c | cut -f2`-${{ matrix.llvm }} main" || true
+        sudo apt-get install -y autoconf cmake gcc g++ libtool gfortran llvm-${{ matrix.llvm }}-dev ninja-build pip
+        sudo python3 -m pip install --upgrade pip setuptools
+        sudo python3 -m pip install lit
+        sudo touch /usr/lib/llvm-${{ matrix.llvm }}/bin/yaml-bench
```

Comment:
This LLVM should not be needed. We need to build Enzyme against the intel llvm

Reply:
We might also be able to build against the fortran compiler.

Reply:
@chelini do you know the ifx folks by chance?

Reply:
> k a look at ifx and it does not seem like it supports llvm plugins. So we don't need to build it. We only need to build against the llvm used to compile opt or lld (whic

@wsmoses no but I can ask around. What do you need?

Reply:
I'm resolving this conversation as we have decided to use standard LLVM for now.